meson.build

```diff
@@ -489,4 +489,8 @@ pkgc.generate(name: 'eigen3',
 
 eigen_dep = declare_dependency(
     include_directories : include_directories('.'),
-)
\ No newline at end of file
+)
+
+[provide]
+eigen3 = eigen_dep
+
```

Comment:
You seem to have accidentally added this to the wrong file. :p

Reply:
ugh... i'm not real smart...  thanks for sanity checking this carefully!